packages/autocomplete-core/src/types/api.ts

```diff
@@ -35,13 +35,11 @@ export interface GetSourcesParams<TItem> extends AutocompleteSetters<TItem> {
 }
 
 interface ItemParams<TItem> {
-  suggestion: TItem;
-  suggestionValue: ReturnType<
-    InternalAutocompleteSource<TItem>['getInputValue']
-  >;
-  suggestionUrl: ReturnType<
-    InternalAutocompleteSource<TItem>['getSuggestionUrl']
+  item: TItem;
+  itemInputValue: ReturnType<
+    InternalAutocompleteSource<TItem>['getItemInputValue']
   >;
+  itemUrl: ReturnType<InternalAutocompleteSource<TItem>['getItemUrl']>;
```

Comment:
why are these prefixed by `item`, when we're already in an item? could inputValue & url be sufficient?

Reply:
They're coming from `getItemInputValue` and `getItemUrl` so it's easier to map the naming to these.